epictrack-web/src/components/proponent/ProponentForm.tsx

```diff
@@ -68,6 +68,10 @@ export default function ProponentForm({ ...props }) {
     reset(ctx.item);
   }, [ctx.item]);
 
+  const setDefaultValues = () => {
+    ctx.setItem({ is_active: true });
```

Comment:
might this overwrite the is_active for an existing item?

Reply:
if there is an existing item it uses the correct value

Reply:
Would something like this work
`  React.useEffect(() => {
    reset({
      ...workFirstNation,
      work_id: ctx.work?.id,
      is_active: true,
    });
  }, [ctx.work?.id]);`